### ISSUE TITLE
It is very tedious to increase/decrease colorSlider on iOS w/ VO

### ISSUE THREAD

**Issue by jnurthen** [adobe/react-spectrum#4101]
# 📝 Feedback

ColorSlider increases/decreases by only 1 on each swipe up/down when using voiceover on iOS. There doesn't seem to be any way to increase this increment. This makes it very tedious to increase/decrease the value.
You can use tap and hold to move the slider but there is no audible feedback on the new value when using this method so it doesn't really work well.

## 💻 Code Sample

https://react-spectrum.adobe.com/react-spectrum/ColorSlider.html

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| [Package name]   | ColorSlider
| Browser          | Safari
| Operating System | iOS 16.3.1

**Comment by dannify** [comment]
Thanks James. Do you have any suggestions on how we can improve this?

**Comment by majornista** [comment]
@jnurthen and @dannify I see this as a VoiceOver on iOS problem. The behavior can be reproduced using any `input[type="range"]` which is used to handle increment and decrement in our Slider, RangeSlider and ColorSlider controls.  VoiceOver does not announce the value changes when a user interacts using tap and hold to move the slider.

**Comment by majornista** [comment]
Experiment with using live-announcer to announce values when dragging.
